turbine/src/cluster_nodes.rs

```diff
@@ -576,23 +599,9 @@ pub(crate) fn get_data_plane_fanout(shred_slot: Slot, root_bank: &Bank) -> usize
             311 => 384,
             _ => DATA_PLANE_FANOUT,
         }
-    } else {
-        DATA_PLANE_FANOUT
     }
 }
 
-fn enable_turbine_fanout_experiments(shred_slot: Slot, root_bank: &Bank) -> bool {
```

Comment:
I sort of liked this old logic more. The effectively dead else leg seems more confusing

Reply:
Which else leg is dead?

I had to change this not to check `disable_turbine_fanout_experiments` twice.

Reply:
nevermind, misread things. I now see that the middle part of the if/else is checking the new `enable_turbine_extended_fanout_experiments` feature

Reply:
And because of that, this point is moot. I now understand why you had to change things.